lib/rest/Twilio.js

```diff
@@ -220,6 +222,7 @@ function Twilio(username, password, opts) {
  * @param {object} [opts.data] - The request data
  * @param {int} [opts.timeout] - The request timeout in milliseconds
  * @param {boolean} [opts.allowRedirects] - Should the client follow redirects
+ * @param {string} [opts.logLevel] - Show debug logs
```

Comment:
This isn't a request-level parameter, but maybe it should be: `logLevel: opts.logLevel || this.logLevel`?

Reply:
Is this not what's happening [below](https://github.com/twilio/twilio-node/pull/618/files#diff-3685a5bef74c26aa2f3cfd3dae3f1a63557c169fab314b0d342698195782a320R271) ? Is the ask here just to change the parameter comment to state `@param {string} [this.logLevel] - Show debug logs`

Reply:
No, what you're doing below is `logLevel: this.logLevel`. You're not using `opts.logLevel` anywhere.